python/HelloWorldTest.py

```diff
@@ -0,0 +1,76 @@
+from selenium import webdriver
+import unittest
+from pathlib import Path
+import os
+from sys import platform as _platform
+
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import re
+
+
+class HelloWorldTests (unittest.TestCase):
```

Comment:
Unnecessary space between Class name and parenthesis.

Reply:
Removed